frontend/components/SearchBar.vue

```diff
@@ -80,6 +79,7 @@
 <script setup lang="ts">
 import { useMagicKeys, whenever } from "@vueuse/core";
 import { ref } from "vue";
+import BaseTooltip from "./tooltip/BaseTooltip.vue"
```

Comment:
Similarly here - no need for the import within Nuxt. We need to import from extensions, but all components are auto loaded 🚀